web/src/components/DatasetSubimageSelector.vue

```diff
@@ -0,0 +1,84 @@
+<template>
+  <div>
+    <v-tooltip
+      v-if="!selectionMode"
+      bottom
+    >
+      <template v-slot:activator="{ on, attrs }">
+        <v-btn
+          icon
+          v-bind="attrs"
+          v-on="on"
+          @click="() => store.commit.setSelectionMode(true)"
+        >
+          <v-icon>mdi-select</v-icon>
+        </v-btn>
+      </template>
+      <span>Select a portion of the data</span>
```

Comment:
```suggestion
      <span>Create a dataset for a region of the image</span>
```
I think the tooltip should at least mention creating a new dataset.